projects/power_selection/src/power_selection.c

```diff
@@ -34,15 +34,32 @@ AdcChannel aux_channels[AUX_ADC_CURRENT_CHANNEL] = { [AUX_ADC_VOLT_CHANNEL] = AD
 static uint16_t s_status = 0;
 static uint16_t s_aux_volt = 0;
 static uint16_t s_aux_temp = 0;
+
+// note: this section may be used as a smoke test, logging has been separated into this function
+/*The typical readings are:
+[0] projects/power_selection/src/power_selection.c:40: AUX Volatge Data: 2500
+[0] projects/power_selection/src/power_selection.c:41: AUX Temp Voltage Data: 2500
+[0] projects/power_selection/src/power_selection.c:42: AUX Temp Resistance Value: 3200.000000
+[0] projects/power_selection/src/power_selection.c:43: AUX Temp Data in C: 53
+*/
```

Comment:
nit: we don't use multi line comments in our codebase, don't really know why, just a legacy thing, but please use single line comments